packages/theme-patternfly-org/scripts/getHtmlWebpackPlugins.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const prerender = require('./prerender');
+const { googleAnalyticsID } = require(`${process.cwd()}/patternfly-docs.config`);
+
+module.exports = (routes, isProd) =>
+  Object.entries(routes).map(([url, { title }]) => 
+    new HtmlWebpackPlugin({
+      template: path.resolve(__dirname, '../template.ejs'),
+      filename: `${url}/index.html`.replace(/^\/+/, ''),
+      title: `PatternFly 4${title ? `- ${title}` : ''}`,
```

Comment:
is this the title for each page? Just a suggestion, but I think a bullet instead of a hyphen would look good here as well, since the title can also contain hyphens. If you hover over a github tab you'll see what I mean :)

Reply:
I just went with what our site currently uses. I'll use `•` instead :)

Reply:
cool, there should also be a space after the 4

Reply:
Fixed!